packages/x-license-pro/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license-pro",
-  "version": "5.17.12",
+  "version": "5.17.13",
```

Comment:
Did something change in the license package? 🤔 
From changelog it does not seem like it did... 🤔 
If it did, we should probably sync it with `data-grid` (package) version.